scss/_patterns_breadcrumbs.scss

```diff
@@ -5,6 +5,7 @@
   .p-breadcrumbs {
     list-style: none;
     margin: 0;
+    overflow-y: auto;
```

Comment:
Why the change in overflow?

Reply:
to make sure the margin-bottom of children is covered by the height of the parent. Otherwise it still applies, but it looks like it doesn't - the background of the example doesn't extend to cover it.

Reply:
This should also be applied to the body, otherwise many components look like they have no margin-bottom, as @anthonydillon pointed out further up in this thread.

Reply:
I don't think we should apply it for the sake of making examples look "better", we would need to apply this to every component in Vanilla and it can have unexpected consequences (scrollbars appearing).

If we want to make margins visible in examples we need to find different solution - maybe by applying some styles to body only in the examples? But that's something for another PR.

I'd remove this overflow from here as it doesn't seem to be needed for correct styling of the component.